src/renderer/components/input/input_validators.ts

```diff
@@ -92,30 +114,26 @@ export const isExtensionNameInstall = inputValidator({
   validate: value => isExtensionNameInstallRegex.isMatch(value),
 });
 
-export const isPath = inputValidator<true>({
+export const isPath = asyncInputValidator({
   debounce: 100,
   condition: ({ type }) => type === "text",
   validate: async value => {
-    try {
-      await fse.pathExists(value);
-    } catch {
-      throw new AsyncInputValidationError(`${value} is not a valid file path`);
+    if (!await fse.pathExists(value)) {
```

Comment:
Looks like old code catch the error throw from `fse.pathExists`, do we still need to catch this exception, or it's handled somewhere else?